hibikase/LineTimingDecorations.cpp

```diff
@@ -65,6 +61,31 @@ SyllableDecorations::SyllableDecorations(const QPlainTextEdit* text_edit, int st
     setVisible(true);
 }
 
+QColor SyllableDecorations::GetPlayingColor() const
+{
+    return QPalette().color(QPalette::Text);
+}
+
+QColor SyllableDecorations::GetNotPlayedColor() const
+{
+    QColor col(0x55, 0x77, 0x55);
```

Comment:
Is the not played color supposed to have been swapped with the played color? This one was `0x77, 0x55, 0x77` before and vice versa.

Reply:
oops yes I forgot to fix that

somewhere along the way with all the times I rewrote this I swapped the colours accidentally, and I noticed this during debugging :p